packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/DecreaseWorkingGroupLeadStake.tsx

```diff
@@ -0,0 +1,118 @@
+import BN from 'bn.js'
+import React, { useEffect } from 'react'
+
+import { InputComponent, InputNumber } from '@/common/components/forms'
+import { Info } from '@/common/components/Info'
+import { AmountButton, AmountButtons, Row, TransactionAmount } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TextInlineMedium, TextMedium } from '@/common/components/typography'
+import { capitalizeFirstLetter } from '@/common/helpers'
+import { useNumberInput } from '@/common/hooks/useNumberInput'
+import { formatTokenValue } from '@/common/model/formatters'
+import { SelectMember } from '@/memberships/components/SelectMember'
+import { useMember } from '@/memberships/hooks/useMembership'
+import { SelectWorkingGroup } from '@/working-groups/components/SelectWorkingGroup'
+import { useWorkingGroup } from '@/working-groups/hooks/useWorkingGroup'
+
+export interface DecreaseWorkingGroupLeadStakeParameters {
+  stakingAmount?: BN
+  groupId?: string
+  workerId?: number
+}
+
+interface DecreaseWorkingGroupLeadStakeProps extends DecreaseWorkingGroupLeadStakeParameters {
+  setStakingAmount: (amount: BN) => void
+
+  setGroupId(groupId: string): void
+
+  setWorkerId(workerId?: number): void
+}
+
+export const DecreaseWorkingGroupLeadStake = ({
```

Comment:
Could add a stories as well? You can see how other steps stories were created.

Reply:
I am not sure that I can add stories for this step. Because it uses worker mocks

Reply:
You can always add them ;)

[https://github.com/Joystream/pioneer/blob/c4f14898a3b7f40af3136920a177a37596817118/packages/ui/src/mocks/components/storybook/MockApolloProvider.tsx#L35-L38](https://github.com/Joystream/pioneer/blob/c4f14898a3b7f40af3136920a177a37596817118/packages/ui/src/mocks/components/storybook/MockApolloProvider.tsx#L35-L38)